frontend/src/app/components/blocks-list/blocks-list.component.html

```diff
@@ -1,29 +1,30 @@
 <app-indexing-progress *ngIf="!widget"></app-indexing-progress>
 
-<div class="container-xl" [class]="widget ? 'widget' : 'full-height'">
+<div class="container-xl" [class]="widget ? 'widget' : 'full-height'" [class]="indexingAvailable ? '' : 'legacy'">
   <h1 *ngIf="!widget" class="float-left" i18n="master-page.blocks">Blocks</h1>
 
   <div class="clearfix"></div>
 
   <div style="min-height: 295px">
     <table class="table table-borderless">
       <thead>
-        <th class="height text-left" [class]="widget ? 'widget' : ''" i18n="latest-blocks.height">Height</th>
-        <th class="pool text-left" [class]="widget ? 'widget' : ''" i18n="mining.pool-name">Pool</th>
-        <th class="timestamp" i18n="latest-blocks.timestamp" *ngIf="!widget">Timestamp</th>
-        <th class="mined" i18n="latest-blocks.mined" *ngIf="!widget">Mined</th>
-        <th class="reward text-right" i18n="latest-blocks.reward" [class]="widget ? 'widget' : ''">Reward</th>
-        <th class="fees text-right" i18n="latest-blocks.fees" *ngIf="!widget">Fees</th>
-        <th class="txs text-right" i18n="dashboard.txs" [class]="widget ? 'widget' : ''">TXs</th>
-        <th class="size" i18n="latest-blocks.size" *ngIf="!widget">Size</th>
+        <th class="height text-left" [class]="widget ? 'widget' : ''" [class]="indexingAvailable ? '' : 'legacy'" i18n="latest-blocks.height">Height</th>
```

Comment:
Maybe it's not possible to combine multiple `[class]` without conflict.

Instead consider:

`[ngClass]="{'widget: widget, 'legacy': !indexingAvailable}"`

Reply:
That fixed it, thanks!